toolkit/src/styles/components/list-item.scss

```diff
@@ -53,4 +53,49 @@ a.cut-list-item {
 
 .cut-list-item .cut-list-item__content {
   cursor: pointer;
+}
+
+.cut-list-item-label {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+.cut-list-item-label .title-row {
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+  align-items: center;
+}
+
+.cut-list-item-label--title {
+  @extend .hds-typography-body-300;
```

Comment:
as mentioned in the other PR, you can apply the class names directly to the elements, instead of using `@extend` (which has a lot of downsides)